libs/_autotools/aqbanking/aqbanking.py

```diff
@@ -64,14 +64,10 @@ def postInstall(self):
                         if craftRoot.endswith("/"):
                             craftRoot = craftRoot[:-1]
                         cmakeFileContents[i] = cmakeFileContents[i].replace(m.group('root'), craftRoot)
-
-                    m2 = re.search("libaqbanking.so", cmakeFileContents[i])
-                    if m2 is not None:
-                        cmakeFileContents[i] = cmakeFileContents[i].replace("lib/libaqbanking.so", "bin/libaqbanking-35.dll")
                 elif CraftCore.compiler.isMacOS:
```

Comment:
we had an issue with symlinks before, is that step needed? a symlink libaqbanking.so  -> libaqbanking.so.ver should exist nowadays.

Reply:
The earlier aqbanking 5.3.8 only had a single line in its cmake which assumed `.so` extension irregardless of compilation platform: `set_and_check(AQBANKING_LIBRARIES "@libdir@/libaqbanking.so"). This is why the blueprint contained a patching logic for both win32 and macOS to lookup the correct extensions. 

The 6.1.2, however, added win32 handling: 

```if(WIN32)
    set_and_check(AQBANKING_LIBRARIES "@libdir@/libaqbanking.dll.a")
else()
    set_and_check(AQBANKING_LIBRARIES "@libdir@/libaqbanking.so.@AQBANKING_SO_EFFECTIVE@")
endif()
```
I assumed this works as expected, so I removed the patching logic for win32 altogether.